src/Components/Cartitem/Cartitem.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import SampleProduct from "../../images/SampleProduct";
+const Cartitem = (img, price) => {
```

Comment:
you must receive an object here
```suggestion
const Cartitem = (product) => {
```

Reply:
the name should be `CardItem`, change the file name and folder too